app/api/server/v1/chat.js

```diff
@@ -360,6 +360,30 @@ API.v1.addRoute('chat.ignoreUser', { authRequired: true }, {
 	},
 });
 
+API.v1.addRoute('chat.getIgnoredUsers', { authRequired: true }, {
+	get() {
+		const { rid } = this.queryParams;
+		if (!rid || !rid.trim()) {
+			throw new Meteor.Error('error-room-id-param-not-provided', 'The required "rid" param is missing.', { method: 'getIgnoredUsers' });
+		}
+
+		const { userId } = this.userId;
```

Comment:
`this.userId` does not contain a `userId` property.

Reply:
So i should use Meteor.userId() instead, right?